web/html/src/components/datetime/FromNow.tsx

```diff
@@ -14,3 +14,7 @@ export const FromNow = (props: Props) => {
   const value = localizedMoment(rawValue).tz(localizedMoment.userTimeZone);
   return <span title={value.toUserString()}>{value.fromNow()}</span>;
 };
+
+export const fromNowStringfied = (value) => {
```

Comment:
Conceptually I'm not too keen on this function since it's essentially the same thing as the component above, except no span wrapper. I think it might make more sense to make `t()` handle simple components too, but I think that's way outside of the scope of anything that this PR addresses. 😄

I think we can rename it as `fromNow` as a compromise since then it's the same thing, component with capital, function with lowercase.

Reply:
@Etheryte good point. Done --> https://github.com/uyuni-project/uyuni/pull/4303/commits/8513993de6dd41df3daada6ffb1abe7095466265